### PR TITLE
Lab 3: "HTML/CSS"

### PR DESCRIPTION
**Pull request by GritsH**
https://gritsh.github.io/itirod-webapp-auction

Originally, i wanted to deploy my project in heroku, but something went wrong, so here we are with my application on github pages

(found out that heroku is used when you have a backend, so i assume i did everything right)

Also, can you please tell me what functions for the fourrth work i need to implement, so i can start as soon as possible

### CODE REVIEW

File: css/addItem.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
+
+.name-label {
+    position: absolute;
```
Comment (katetushkan) [inline css/addItem.css:3]: why absolute?

Comment (katetushkan) [reply css/addItem.css:3]: don't use percents for such a values

File: css/addItem.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    top: 20%;
+    left: 5%;
+}
+
+.item-name-input {
+    position: absolute;
```
Comment (katetushkan) [inline css/addItem.css:9]: same here. why absolute?

**Review by katetushkan — CHANGES_REQUESTED** [review]
Let's have a call if u are OK.

**Comment by GritsH** [comment]
Yeah, let's have a call, just tell me when and where so i can be prepared 😰

пн, 28 мар. 2022 г. в 14:39, Katarina Tushinskaya ***@***.***

**Comment by katetushkan** [comment]
U don't need to be prepared, I'll just explain u why absolute is not an option. March the 30th at 11-40 will be fine?

File: css/auctionRoom.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
+.item-info_ {
```
Comment (katetushkan) [inline css/auctionRoom.css:1]: Если хочешь чтобы было в центре, тогда используй text-align, а не маржин

File: css/createLots.css [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+.auction-step-input {
+    width: 31%;
+    height: 7%;
+}
+
+
+.starting-date-input {
+    width: 31%;
+    height: 7%
```
Comment (katetushkan) [inline css/createLots.css:20]: лучше rem, a не %

File: html/myAuctions.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    <div class="scroll" style="grid-row: span 2">
+        <div class="lot-preview">
+            <label class="item-info">Name: </label><br>
+            <label class="item-info">Description: </label><br>
+            <label class="item-info">Starting price: </label><br>
+            <label class="item-info">Auction step: </label><br>
+            <hr style="width: 95%">
+
+            <label class="lot-info">Will begin at: </label><br>
+            <label class="lot-info">Current amount of participants: </label><br>
```
Comment (katetushkan) [inline html/myAuctions.html:25]: Если нет `<input>`, то не может быть и  `<label>`. Надо заменить на `<p>` например

File: html/myAuctions.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    </div>
+</header>
+<div class="grid-container" style="margin-top: 10px">
+    <div class="scroll" style="grid-row: span 2">
+        <div class="lot-preview">
+            <label class="item-info">Name: </label><br>
+            <label class="item-info">Description: </label><br>
+            <label class="item-info">Starting price: </label><br>
+            <label class="item-info">Auction step: </label><br>
+            <hr style="width: 95%">
```
Comment (katetushkan) [inline html/myAuctions.html:22]: вот это можно убрать, обернуть группу в контейнер, и сделать его flex

**Review by katetushkan — COMMENTED** [review]
Надо исправить то, что я написала и это будет на 8. Если хочешь больше - надо добавить стили  focus/ hover/ disabled на кнопки и линик.

**Comment by GritsH** [comment]
I’m doing some changes with styles you told me about right now 
If you can wait for about half an hour or so, you can check them 😰

**Comment by katetushkan** [comment]
Well, I'm sorry, but that's still an 8( u need look better in flex positioning and mobile version. It's working but it's not great. Let me know if u are okay with 8 or u wanna fix. 
Screenshot 2022-04-11 at 15 13 12
.

**Comment by GritsH** [comment]
Well, whatever 
I’ve missed deadline for the 4th work, so that doesn’t really matter 
I agree with 8

**Comment by katetushkan** [comment]
> Well, whatever
> I’ve missed deadline for the 4th work, so that doesn’t really matter
> I agree with 8

`8` is urs, don't be upset about deadline, a lot of ur group mates haven't started a 1st work.

**Pull request merged**